go/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Move the go directory into $PREFIX/ so it can be built in its install location.
+cd .. && mv go $PREFIX/ && cd $PREFIX/go
```

Comment:
Take a look at my first commit if you want to see another way, in which I tried to install it into `$PREFIX`. From what I can tell, it seemed to work, but I have no idea what might break as it is non-standard.